docs/requests.md

```diff
@@ -60,8 +60,8 @@ For example: `request.path_params['username']`
 
 #### Client Address
 
-The client's remote address is exposed as a named two-tuple `request.client`.
-Either item in the tuple may be `None`.
+The client's remote address is exposed as a named two-tuple `request.client` (or `None`).
+`request.client` may be `None` but each item in the tuple will not be `None`.
```

Comment:
I meant with this being removed
```suggestion
```

Reply:
My bad, I forgot that I had still to remove the line